lms/djangoapps/grades/models.py

```diff
@@ -517,7 +517,7 @@ class Meta(object):
 
     # Information about the course grade itself
     percent_grade = models.FloatField(blank=False)
-    letter_grade = models.CharField(u'Letter grade for course', blank=False, max_length=255)
+    letter_grade = models.CharField(u'Letter grade for course', blank=False, null=False, max_length=255)
```

Comment:
Nit: No need to include default value for `null`.  Since you're no longer changing the model, might as well keep it untouched.